ckcp/openshift_dev_setup.sh

```diff
@@ -171,25 +172,6 @@ install_openshift_gitops() {
 }
 
 install_cert_manager() {
```

Comment:
Rename func to `check_cert_manager`, and reinstate `echo -n "  - openshift-cert-manager-operator: "`.

Reply:
rename and reinstated.